runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java

```diff
@@ -176,6 +184,11 @@ public void onGameTick(GameTick tick)
 			barOverlay.onTick();
 		}
 
+		if (config.replaceOrbText() && isAnyPrayerActive())
+		{
+			setPrayerOrbText(getEstimatedTimeRemaining(true));
+		}
```

Comment:
Given your comment regarding the orb text updating after script 82 runs, maybe this should be moved to an `onPostScriptFired` subscriber?

Reply:
Yes that is probably a good idea

Reply:
On second thought, this would cause the remaining time to not appear when toggling individual prayers until a point is drained, so that's not great. That requires this code being run each game tick, so this should be fine after all.